File: Iniciante/URI 2786.ml
let l = read_int();;
let c = read_int();;
let lajes1 = (l * c) + ((l - 1) * (c - 1));;
let lajes2 = ((l - 1) * 2) + ((c - 1) * 2);;
Printf.printf "%d\n" (lajes1);
Printf.printf "%d\n" (lajes2);

File: Iniciante/URI 2786.dart
import 'dart:io';

void main() { 
  int L = int.parse(stdin.readLineSync());
  int C = int.parse(stdin.readLineSync());
  int lajes1 = (L * C) + ((L - 1) * (C - 1));
  int lajes2 = ((L - 1) * 2) + ((C - 1) * 2);
  print(lajes1);
  print(lajes2);
}